espnet2/bin/tts_inference.py

```diff
@@ -306,7 +306,7 @@ def from_pretrained(
         return Text2Speech(**kwargs)
 
 
-@typechecked
+# @typechecked
```

Comment:
Please refer to the change (and also update the output_dir type accordingly as tts2_inference.py)